palette/src/convert.rs

```diff
@@ -391,7 +391,7 @@ where
 ///         0.7353569830524495,
 ///         0.5370987304831942,
 ///     ];
-///     let hsv = Bgr::from_raw_slice(&buffer)[1].into();
+///     let hsv: Hsv<palette::encoding::Srgb, _> = Bgr::from_raw_slice(&buffer)[1].into();
```

Comment:
Wasn't just `Hsv` enough? `Srgb` should be the default.

Reply:
yeah strangely enough, just `Hsv` didn't seem to be enough :(

Reply:
That's odd. Oh well. In that case can you add `Srgb` to the import at the top of the example? That would make this part a bit less noisy... I don't know if you tried it, but you may be able to skip the `_` at least, unless I'm forgetting something.

Reply:
That would conflict with `palette::Srgb` imported above - do you have a preferred alias for the `Srgb` encoding? (e.g. `eSrgb` or similar?)

I have tried without the `_` and that doesn't seem to be enough.

Reply:
Ugh, good job on me for not reading it all. It's also `f64`, not the default `f32`, so that's probably why `_` has to be there.

Uhm... I usually prefer to spell it out if I can... You could put a `self` in the encoding import and at least shorten it down to `encoding::Srgb`, I guess. :) If that makes a difference. I'm just trying to put the focus on the right hand expression, but it's not that important.

Reply:
Oh, you can create an alias like `type Hsv64 = Hsv<encoding::Srgb, f64>;`!